controllers/datadogagent/feature/cws/feature.go

```diff
@@ -0,0 +1,180 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cws
+
+import (
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v1alpha1"
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v2alpha1"
+	apiutils "github.com/DataDog/datadog-operator/apis/utils"
+
+	apicommon "github.com/DataDog/datadog-operator/apis/datadoghq/common"
+	apicommonv1 "github.com/DataDog/datadog-operator/apis/datadoghq/common/v1"
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/feature"
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/object/volume"
+)
+
+func init() {
+	err := feature.Register(feature.CWSIDType, buildCWSFeature)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func buildCWSFeature(options *feature.Options) feature.Feature {
+	cwsFeat := &cwsFeature{}
+
+	return cwsFeat
+}
+
+type cwsFeature struct {
+	configMapConfig       *apicommonv1.ConfigMapConfig
+	configMapName         string
+	syscallMonitorEnabled bool
+}
+
+// Configure is used to configure the feature from a v2alpha1.DatadogAgent instance.
+func (f *cwsFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.RequiredComponents) {
+	if dda.Spec.Features != nil && dda.Spec.Features.CWS != nil && apiutils.BoolValue(dda.Spec.Features.CWS.Enabled) {
+		cws := dda.Spec.Features.CWS
+
+		if apiutils.BoolValue(cws.SyscallMonitorEnabled) {
+			f.syscallMonitorEnabled = true
+		}
```

Comment:
The `if` could be simplified: `f.syscallMonitorEnabled = apiutils.BoolValue(cws.SyscallMonitorEnabled)`